crates/bevy_app/src/ci_testing.rs

```diff
@@ -20,18 +22,29 @@ fn ci_testing_exit_after(
     if let Some(exit_after) = ci_testing_config.exit_after {
         if *current_frame > exit_after {
             app_exit_events.send(AppExit);
+            info!("Exiting after {} frames. Test successful!", exit_after);
         }
     }
     *current_frame += 1;
 }
 
 pub(crate) fn setup_app(app: &mut App) -> &mut App {
-    let filename =
-        std::env::var("CI_TESTING_CONFIG").unwrap_or_else(|_| "ci_testing_config.ron".to_string());
-    let config: CiTestingConfig = ron::from_str(
-        &std::fs::read_to_string(filename).expect("error reading CI testing configuration file"),
-    )
-    .expect("error deserializing CI testing configuration file");
+    #[cfg(not(target_arch = "wasm32"))]
+    let config: CiTestingConfig = {
+        let filename = std::env::var("CI_TESTING_CONFIG")
+            .unwrap_or_else(|_| "ci_testing_config.ron".to_string());
+        ron::from_str(
+            &std::fs::read_to_string(filename)
+                .expect("error reading CI testing configuration file"),
+        )
+        .expect("error deserializing CI testing configuration file")
+    };
+    #[cfg(target_arch = "wasm32")]
+    let config: CiTestingConfig = {
+        let config = include_str!("../../../ci_testing_config.ron");
```

Comment:
Again, should this be a custom property on document/window?

Reply:
I don't think you can `include_str` something not a string literal?

Reply:
I mean sure? But you can read a string which isn't a string literal. We don't do anything special with this string that requires it to be statically inlined in the WASM blob.

Reply:
I'm not sure I understand what you mean? You don't have access to the file system in wasm.

Reply:
Why would we need file system access to read the config values?

They would be assigned onto the window object as a string or object

Reply:
I looked a little into how to add those to to the javascript window object, it could work but mean that the file would be read from typescript, and I don't want to start reading ron files in typescript. And I would prefer to keep the same file format as the other example runs

Reply:
This should be fixed somehow but I would rather get the benefits of running wasm examples sooner than later and this doesn’t really seem trivial to fix in a good way.